cardano-api/ChangeLog.md

```diff
@@ -27,6 +27,13 @@
 - **Breaking change** - `queryExpr` to return `IO (Either UnsupportedNtcVersionError a)` instead of `IO a`.
   ([PR4788](https://github.com/input-output-hk/cardano-node/pull/4788))
 
+- **Breaking change** - For performance reasons, `evaluateTransactionFee` to take a
+  `Ledger.PParams (ShelleyLedgerEra era)` argument instead of `ProtocolParameters`
+  ([PR4903](https://github.com/input-output-hk/cardano-node/pull/4903))
+
+- New type `BundledProtocolParameters` and new functions `bundleProtocolParams` and `unbundleProtocolParams`.
```

Comment:
Let's merge this with the breaking change above. It could just be the following sentence.

Reply:
Done!